src/model/ControllerMechanicalVentilation.cpp

```diff
@@ -103,9 +109,28 @@ namespace model {
     }
 
     std::string ControllerMechanicalVentilation_Impl::systemOutdoorAirMethod() const {
-      boost::optional<std::string> value = getString(OS_Controller_MechanicalVentilationFields::SystemOutdoorAirMethod, true);
-      OS_ASSERT(value);
-      return value.get();
+      boost::optional<std::string> result;
+      const auto value = getString(OS_Controller_MechanicalVentilationFields::SystemOutdoorAirMethod, true);
+      if (value && !value->empty()) {
+        result = value;
```

Comment:
(I would have used `getString(XXX, true, true)` (signature is `boost::optional<std::string> getString(unsigned index, bool returnDefault = false, bool returnUninitializedEmpty = false) const;`) but this will do as well)